lib/utils/regexes.mjs

```diff
@@ -0,0 +1,11 @@
+import { nestingSupportedAtKeywords } from '../reference/atKeywords.mjs';
+
+export const atRulesRegexes = {
```

Comment:
[suggestion] This is quite a personally opinion and just bike-shedding, but I feel `s` at the end of `atRules` is a bit less readable since the name already ends with `s` in `Regexes`.

So, how about removing a doubled `s` like this?

```suggestion
export const atRuleRegexes = {
```

This is just my opinion, so if you are against it, you don't need to address it. 😃